### ISSUE TITLE
Capture exception as resolved

### ISSUE THREAD

**Issue by jvenezia** [getsentry/sentry#6324]
Following this issue https://github.com/getsentry/raven-ruby/issues/758

I would love to be able to send exceptions to sentry and mark then directly as resolved:
`Raven.capture_exception(exception, resolved: true)`

According to @nateberkopec it is not possible in the current API.

Can you consider adding such feature? Thanks!

**Comment by dcramer** [comment]
I dont understand the use case. If you're simply not wanting notified about it, 'resolved' wouldn't be the right term. I could see it as being like "dont notify" or "ignore", but I'd want to know more. It'd be like "I want to create a new issue on GitHub, but automatically close it".

**Comment by jvenezia** [comment]
Yes probably 'resolved' is not the right term. 'muted' or 'ignored' would be more accurate.

My use-case is that I have exceptions that I want to mute because they will be retried. Most of the time they are exceptions raised by external services, it can happen that it fails (too many requests, service unavailable or whatever). So I want to log them, but not be notifier.

It is not general to a particular exception. Sometimes I want to mute them, sometimes a want to be notified. So a rule making it always silent would not work.

Today we log them in another place, but we would love to log them in Sentry too.
Sentry is awesome to notify exceptions, display them the right way and manage them etc.
What I miss today is a way to use sentry to log exceptions that I am aware of but I deliberately want to mute.
Basically I want to get the benefit of having my exceptions on sentry (because of the UI history, management etc) without the notification part.

Does that make sense? Happy to share more details if needed.

**Comment by MaxBittker** [comment]
ah, that does make sense @jvenezia. However, I think that setting `level:info` and altering your alert rules & saved searches could give you a pretty good solution for this behavior. I'm going to go out on a limb and close this issue, but let me know if you disagree  

https://docs.sentry.io/clientdev/attributes/#optional-attributes

**Comment by dcramer** [comment]
Agreed that using info for now is great and will future proof things.

There’s a day when sentry will let you capture a lot more events without creating an “issue” out of each one. 

Thanks for filling in the details!

**Comment by jvenezia** [comment]
Thanks @MaxBittker @dcramer 

This could work, but I can't find how to disable slack/email notifications for issues with `level:info`.

Also can I set the default issues filter in the UI to no show `level:info` exceptions? like `is:unresolved level:not:info` ?

**Comment by dcramer** [comment]
@jvenezia we dont provide any great ways to change defaults like that, but what you could do is this:

1. Navigate to the project -> settings -> alerts -> rules
2. Change the default rule to have a new condition of 'An events level ...'
3. If you want to send any kind of notifications for lower level events, you can create a second rule and limit which services it goes to
